core/src/test/resources/sql/functions.iq

```diff
@@ -464,4 +464,19 @@ limit 3;
 
 !ok
 
+# SUBSTRING
+-- returns 'null'
+select SUBSTRING(NULL FROM 1 FOR 2);
```

Comment:
can these test cases in quidem file be put together? I'm not sure if this works for later test cases

Reply:
yes, they can.